ext/configuration.c

```diff
@@ -156,6 +157,51 @@ static bool dd_parse_tags(zai_str value, zval *decoded_value, bool persistent) {
     return true;
 }
 
+static bool dd_parse_http_error_statuses(zai_str value, zval *decoded_value, bool persistent) {
```

Comment:
Is these a reason to use the custom parser at all? The default parser for SET should cover all use cases here, I believe.